src/NuGet.Core/NuGet.DependencyResolver.Core/ResolverUtility.cs

```diff
@@ -482,7 +482,7 @@ private static GraphItem<RemoteResolveResult> CreateUnresolvedResult(LibraryRang
                 Data = new RemoteResolveResult()
                 {
                     Match = match,
-                    Dependencies = Enumerable.Empty<LibraryDependency>()
+                    Dependencies = new List<LibraryDependency>()
```

Comment:
Instead, can we use null as the handshake here? This moved from no allocations to two allocations (List + Array underneath).

Reply:
I think we can just share a list. 

`List<T>(0)` references a static.